solutions/azure-automation-state-configuration/azuredeploy.bicep

```diff
@@ -0,0 +1,620 @@
+targetScope = 'resourceGroup'
+
+/*** PARAMETERS ***/
+
+@description('The location of all resources')
+param location string = resourceGroup().location
+
+@description('Username for both the Linux and Windows VM. Must only contain letters, numbers, hyphens, and underscores and may not start with a hyphen or number.')
+param adminUserName string
+
+@secure()
+@description('Password for both the Linux and Windows VM. Password must have 3 of the following: 1 lower case character, 1 upper case character, 1 number, and 1 special character.')
+param adminPassword string
+
+@description('Email to send not complient alerts')
+param emailAddress string
+
+@description('Windows VMs to be created')
+param windowsVMCount int = 1
+
+@description('Linux VMs to be created')
+param linuxVMCount int = 1
+
+@description('SKU of the Vms')
+param vmSize string = 'Standard_DS1_v2'
+
+@description('Windows VMs configurations')
+param windowsConfiguration object = {
+  name: 'windowsfeatures'
+  description: 'A configuration for installing IIS.'
+  script: 'https://raw.githubusercontent.com/mspnp/samples/main/solutions/azure-automation-state-configuration/scripts/windows-config.ps1'
+}
+
+@description('Linux VMs configurations')
+param linuxConfiguration object = {
+  name: 'linuxpackage'
+  description: 'A configuration for installing Nginx.'
+  script: 'https://raw.githubusercontent.com/mspnp/samples/main/solutions/azure-automation-state-configuration/scripts/linux-config.ps1'
+}
+
+@description('Virtual Network address space')
+param addressPrefix string = '10.0.0.0/16'
```

Comment:
this is hardcoded in connected and others. Please let's follow that convention in here as well.

Reply:
done